ReactAndroid/src/main/java/com/facebook/react/devsupport/JSDebuggerWebSocketClient.java

```diff
@@ -53,7 +50,7 @@
   private @Nullable JSDebuggerCallback mConnectCallback;
   private final AtomicInteger mRequestID = new AtomicInteger();
   private final ConcurrentHashMap<Integer, JSDebuggerCallback> mCallbacks =
-      new ConcurrentHashMap<>();
+    new ConcurrentHashMap<>();
 
   public void connect(String url, JSDebuggerCallback callback) {
     if (mHttpClient != null) {
```

Comment:
Don't think so, this prevents anyone from connecting while it's already connected